src/components/AdminPage.jsx

```diff
@@ -137,6 +157,48 @@ function AdminPage() {
           </table>
         </div>
       </div>
+
+      {chooseCopyModus && (
+        <div className="inventory-container">
+          <form
+            onSubmit={(e) => {
+              e.preventDefault();
```

Comment:
Waarom hier een preventDefault nodig? Is deze render niet al alleen bij de submit button?